packages/big-design/src/components/Panel/Panel.tsx

```diff
@@ -15,22 +17,56 @@ interface PrivateProps {
   forwardedRef: Ref<HTMLDivElement>;
 }
 
-export interface PanelAction extends Omit<ButtonProps, 'children'> {
+export interface PanelActionProps extends Omit<ButtonProps, 'children' | 'mobileWidth'> {
   text?: string;
 }
 
+export interface PanelDropdownProps extends Omit<DropdownProps, 'toggle'> {
+  toggle: PanelActionProps;
+}
```

Comment:
why not just stick to `DropdownProps` entirely? What benefit are we looking for by overwriting the concept of the `toggle`?